frontend/src/components/Register/Register.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import c from "./reg.module.css";
+import axiosInstance from "../../utils/axios";
+
+const Register: React.FC = () => {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState("");
+  const [success, setSuccess] = useState("");
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    try {
+      const response = await axiosInstance.post("/auth/register", {
```

Comment:
Mogli bi napravit jedan centralni api file iz kojeg ćemo onda pozivat axios. Ovako nešto 
![image](https://github.com/user-attachments/assets/4c36fa4a-4e2d-44cb-830b-1986815bdf5f)

Reply:
To ti je ovaj axiosInstance u tome file-u, ja san ga bia bazicno postavia, ali tu samo ima jos za dodat stvari